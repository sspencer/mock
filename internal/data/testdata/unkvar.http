### Unknown var
# @what is this
PUT /random/var/name
