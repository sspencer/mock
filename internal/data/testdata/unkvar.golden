unrecognized variable, line 2: # @what is this